libs/libtatsu/Makefile

```diff
@@ -0,0 +1,71 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libtatsu
+PKG_VERSION:=1.0.3
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=https://github.com/libimobiledevice/libtatsu/releases/download/$(PKG_VERSION)
+PKG_HASH:=4f69452d23e50e0ffbe844110e6ab6a900d080e051fbda3b7d595f679dee9bc5
+
+PKG_MAINTAINER:=Rosen Penev <rosenp@gmail.com>
+PKG_LICENSE:=LGPL-2.1-or-later
+PKG_LICENSE_FILES:=COPYING
+
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/libtatsu/Default
+  TITLE:=Handle communication with Apple's Tatsu Signing Server (TSS)
+  URL:=https://www.libimobiledevice.org/
+  SUBMENU:=libimobiledevice
+endef
+
+define Package/libtatsu/Default/description
+  libtatsu is part of the libimobiledevice project.
+  The main purpose is to allow creating TSS request payloads,
+  sending them to Apple's TSS server, and retrieving and processing the
+  response. The code was originally part of idevicerestore and has been split
+  out to its own library, while obviously idevicerestore will remain the main
+  consumer.
+endef
+
+define Package/libtatsu
+  $(call Package/libtatsu/Default)
+  SECTION:=libs
+  CATEGORY:=Libraries
+  DEPENDS:=+libcurl +libplist
+  LICENSE:=LGPL-2.1-or-later
+  LICENSE_FILES:=COPYING
+endef
+
+define Package/libtatsu/description
+  $(call Package/libtatsu/Default/description)
+endef
```

Comment:
Why this is done like this?

Reply:
Because I followed an existing pattern: I copied the `Makefile` from `libimobiledevice-glue` and adapted it for the new `libtatsu` library. I don't have a good experience with the make files, so any suggestions are welcome. Last time I added `libimobiledevice-glue`, it was accepted, so if you like, once some changes are suggested, I can also apply them there.

Reply:
Unfortunately, thats also wrong and it should not be accepted like that. There is no point having "Default" and then call it. Copying something without knowing what it does, whats it for, its not good approach, though.

Reply:
I figured that the "Default" is used for convenience in other Makefiles that define multiple packages. I applied the changes you suggested.